src/NLog/Layouts/JSON/JsonLayout.cs

```diff
@@ -358,7 +369,10 @@ private void BeginJsonProperty(StringBuilder sb, string propName, bool beginJson
 
         private void CompleteJsonMessage(StringBuilder sb)
         {
-            sb.Append(SuppressSpaces ? "}" : " }");
+            if (IndentJson)
+                sb.AppendLine();
+            
+            sb.Append((IndentJson || SuppressSpaces) ? "}" : " }");
```

Comment:
Do not think you should check IdentJson here also, since SuppressSpaces is false by default.